project/about.md

```diff
@@ -0,0 +1,100 @@
+# About Pacifica
+
+## History
+
+Pacifica was created from a collaboration in the
+[United States National Laboratory](https://en.wikipedia.org/wiki/United_States_Department_of_Energy_national_laboratories)
+system. The Environmental Molecular Sciences Laboratory ([EMSL](https://www.emsl.pnnl.gov))
+collaborated with its host institution Pacific Northwest National Laboratory
+([PNNL](https://www.pnnl.gov)) to help manage the scientific data generated
+by researchers.
+
+EMSL is a [user facility](https://www.energy.gov/science/science-innovation/office-science-user-facilities)
+sponsored by the Department of Energy ([DOE](https://www.energy.gov))
+Office of Science ([here](https://www.energy.gov/science)) Biological and
+Environmental Research ([BER](https://www.energy.gov/science/ber/biological-and-environmental-research)).
+
+Pacifica was created by EMSL and PNNL in response to 
+[DOE Data Management Policy](https://www.energy.gov/datamanagement/doe-policy-digital-research-data-management)
+to manage open research data to make it more transparent. EMSL and
+PNNL both have similar administrative problems in that a single
+project can not shoulder the burden of meeting the policy on its
+own. This brought both EMSL and PNNL to the table to collaborate on
+developing an institutional data management system for open research
+science.
+
+## The Data Management Challenge
+
```

Comment:
The challenge is defined within the nexus of a large institution's operating model, and the data policies the institution is required to follow.

Reply:
Done!